graphql_api/helpers/connection.py

```diff
@@ -68,6 +68,38 @@ def page_info(self, *args, **kwargs):
         }
 
 
+class BetterCursorPaginator(CursorPaginator):
```

Comment:
Nice work.

Reply:
@joseph-sentry Can we update the name to DictCursorPaginator or something similarly less ambiguous